lib/nexpose/vulnerability.rb

```diff
@@ -30,7 +30,7 @@ def supported_tags
         :references, :tags,
 
         # evidence tag
-        :details
+        :details, :name
```

Comment:
what's this #name here? It seems unrelated to the PR?

Reply:
Thanks for catching this! Looks like this was due to my lack of git skillz as it wasn't supposed to be in here. I just pushed a commit to fix it.